docs/pages/enterprise/workflow/ssh-approval-slack.mdx

```diff
@@ -141,7 +158,7 @@ compile these plugins from [source](https://github.com/gravitational/teleport-pl
 <Tabs>
 <TabItem label="Download">
   ```bash
-  $ curl -L https://get.gravitational.com/teleport-access-slack-v(=teleport.version=)-linux-amd64-bin.tar.gz
```

Comment:
This addresses: https://github.com/gravitational/teleport-plugins/issues/229

Also, `curl` no longer throws the same error but we should bring this in line with the rest of the docs. Wasn't sure if there was a specific reason we were using this here (since it uses the -L redirect flag).